design/proposals/spike-cli-dockerless.md

```diff
@@ -42,7 +42,21 @@ The CLI design should answer the following requirements:
 
 *Core functionality*
 
-1. We should support `init`, `authn` (`login`, `authenticate`),  `policy (load)`, and `fetch secrets` functionalities. Open for discussion: `list`, `show, ` `rotate_api_key` (`host` / `user`).
+We should support:
+
+​	Configure connection to Conjur (know as `init`, `authn` (`login`, `authenticate`))
+
+​	Load policy (known as `policy <verb>`)
+
+​	Get/Set secret values (known as `variable values <var_id>` functionalities. 
+
+Open for discussion: 
+
+​	List all resources that I have "read" permissions on (known as `list`)
+
+​	Show resource metadata/annotations (known as `show`)
+
+​	Rotate user and host API key (known as `rotate_api_key`)
```

Comment:
Should "Check permissions on a resource for a given user/host" also be here? I use this in troubleshooting integrations a lot.

Reply:
known as `resource permitted_roles`

Reply:
which one is this in reference to? Rn it seems like you are directly referencing rotate_api_key?

Reply:
I am suggesting a new addition to this list, I just added the comment on the last item